src/FunFair.CodeAnalysis.Tests/ProhibitedMethodsDiagnosticsAnalyzerTests.cs

```diff
@@ -401,5 +401,31 @@ void Test(FunFair.Common.Data.ISqlServerDatabase sqlServerDatabase, string sql)
 
             return this.VerifyCSharpDiagnosticAsync(source: test, expected);
         }
+
+        [Fact]
+        public Task RemoteIpAddressIsBannedAsync()
+        {
+            const string test = @"
+    namespace ConsoleApplication1
+    {
+        class TypeName
+        {
+            void Test()
+            {
+                System.Net.IPAddress connectionRemoteIpAddress = new Microsoft.AspNetCore.Http.DefaultHttpContext().Connection.RemoteIpAddress;
+            }
+        }
+    }";
+
+            DiagnosticResult expected = new DiagnosticResult
+                                        {
+                                            Id = "FFS0026",
+                                            Message = @"Use RemoteIpAddressRetriever",
+                                            Severity = DiagnosticSeverity.Error,
+                                            Locations = new[] { new DiagnosticResultLocation(path: "Test0.cs", line: 8, column: 66) }
+                                        };
+
+            return this.VerifyCSharpDiagnosticAsync(source: test, expected);
```

Comment:
You need to reference the assemblies for  System.Net.IPAddress && the Microsoft.AspNetCore.Http.DefaultHttpContext().Connection.RemoteIpAddress as parameters to 
VerifyCSharpDiagnosticAsync rather than making them global